README.md

```diff
@@ -58,6 +60,8 @@ If you are interested in working on the implementation for an 'active' RFC, but
 
 ## Reviewing RFCs
 
-We try to make sure that any RFC that we accept is accepted at a team meeting. Every accepted feature should have a core team champion, who will represent the feature and its progress.
+We try to make sure that any RFC that we accept is discussed at a team meeting. Every accepted feature should have a core team champion, who will represent the feature and its progress.
+
+An RFC is considered as accepted if it receives at least 4 approving reviews, and no rejection from core team members. If the RFC is proposed by a core team member, this number is lowered to 3.
```

Comment:
Currently, we have a 2-:heavy_check_mark: rule for PRs regardless of who opens the PR.

I think that we should have a higher number here since RFCs are bigger and have more impact. I chose 4 because we are more than 4 in the team, and thus it will make it possible for someone us to "abstain" in case we don't fully agree with the RFC but we don't feel like blocking it.

If the number of active team members decreases, we might need to change this 
threshold.